index.html

```diff
@@ -144,8 +145,8 @@ <h2>Introduction</h2>
         difficulty of scaling user choices by providing a way to universally signal to all website
         publishers, through an HTTP header
         or the DOM, a person's assertion of their applicable rights to prevent the sale of their data,
-        the sharing of their data with third parties, and the use of their data for cross-site targeted
-        advertising. This signal allows users to take advantage of specific provisions in some of these
+        the sharing of their data with third parties, and the use of their data for [=cross-organization targeted
+        advertising=]. This signal allows users to take advantage of specific provisions in some of these
```

Comment:
```suggestion
        the sharing of their data with third parties, including the use of their data for [=cross-party targeted
        advertising=]. This signal allows users to take advantage of specific provisions in some of these
```
s/organization/party/, and make clear that the ad targeting falls under the scope of sharing.